sql-statements/sql-statement-admin-checksum-table.md

```diff
@@ -6,13 +6,27 @@ category: reference
 
 # ADMIN CHECKSUM TABLE
 
-The `ADMIN CHECKSUM TABLE` statement calculates a CRC64 checksum for the data and indexes of a table. This statement is used by programs such as TiDB Lightning to ensure that import operations have completed successfully.
+The `ADMIN CHECKSUM TABLE` statement calculates a CRC64 checksum for the data and indexes of a table.
+
+The [checksum](/tidb-lightning/tidb-lightning-glossary.md#checksum) is calculated over the table data and properties like the `table_id`. This means that two tables with the same data but a different table_id will get different checksums.
+
+<CustomContent platform="tidb">
+
+This statement is used to calculate a checksum that can be compared with the checksum that [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) has calculated locally. This statement is usually executed by [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md).
+
+</CustomContent>
+
+<CustomContent platform="tidb-cloud">
+
+This statement is used to calculate a checksum that can be compared with the checksum that TiDB Lightning has calculated locally. This statement is usually executed by TiDB Lightning.
```

Comment:
@D3Hunter, would you please check whether `ADMIN CHECKSUM TABLE` is also executed by [`IMPORT INTO`](https://docs.pingcap.com/tidb/dev/sql-statement-import-into) before and after the import by default? Thanks.

Reply:
yes, `import-into` will execute it after import

lightning doesn't use it normally

Reply:
@dveeden PTAL, thanks

Reply:
Addressed in ad55fe277a18677cb495bff7b889bdfb649c3427